docs/book/src/user/quick-start.md

```diff
@@ -175,11 +175,29 @@ and `kubeadm` control-plane providers.
 Depending on the infrastructure provider you are planning to use, some additional prerequisites should be satisfied
 before getting started with Cluster API. See below for the expected settings for common providers.
 
+
```

Comment:
nit: extra blank line should not be added.

Reply:
removed